### DISCUSSION TITLE
Some questions about the WSL instance MAC address in order to apply for a AIE build license.

### DISCUSSION THREAD

**Discussion by ngdymx** [AMDResearch/Riallto#18] [Q&A] [answered]
Hi,

I am now in the process of installing Riallto and trying to build the AIE build license. 

Following the tutorial, I require the WSL instance MAC address to apply for an AIE build license. To get the WSL MAC address, I need to run  the following command in Powershell:
"wsl -d Riallto ip -brief link show eth0".

However,  I always get a different MAC address when running the command. Is that okay to apply for the license by choosing any of them?  How can I get a fixed MAC address?

Would anyone be able to help me solve this issue? Thank you very much!

**Comment by mariodruiz** [comment, marked as answer]
> Is that okay to apply for the license by choosing any of them?

Yes.

**Reply by ngdymx** [reply]
I got it. Thanks a lot.

**Reply by mariodruiz** [reply]
If this solved your question, please mark as answered.
